spec/features/world_index_spec.rb

```diff
@@ -1,76 +1,95 @@
 require "integration_spec_helper"
 
 RSpec.feature "World index page" do
-  before do
-    stub_content_store_has_item("/world", GovukSchemas::Example.find("world_index", example_name: "world_index"))
-    visit "/world"
-  end
+  shared_examples "world index page" do
+    scenario "returns 200 when visiting world index page" do
+      expect(page.status_code).to eq(200)
+    end
```

Comment:
**question:** usefulness of tests

Again, maybe it's the style, but it feels like the feature and controller tests are doing almost the same thing: stubbing a response then checking the status code is 200. Presumably the under-the-hood setup will be a bit different between the two test types, but I wonder if one or the other could be more useful and/or require less setup

Reply:
We've deleted the controller spec that was only testing the 200 response